src/components/Location/component.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Jumbotron, Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+import { Message } from '../';
+import { Map, Marker, InfoWindow } from '../../containers';
+import './styles.css';
+
+export default class Location extends Component {
+  bindEvents = () => {
+    const { bindMapEvents } = this.props;
+    const markerOnClick = (props, marker) => this.props.renewMapInfo({ visible: true, marker });
+    const mapOnClick = () => this.props.mapInfo.visible
+      ? this.props.renewMapInfo({ visible: false })
```

Comment:
this.props.renewMapInfo, this.props.mapInfo, this.props.renewMapInfo could be also destructured